beacon_node/lighthouse_network/src/rpc/mod.rs

```diff
@@ -270,8 +271,10 @@ where
         );
 
         // NOTE: this is needed because PeerIds have interior mutability.
-        let peer_repr = peer_id.to_string();
-        let log = self.log.new(slog::o!("peer_id" => peer_repr));
+        let log = self
+            .log
+            .new(slog::o!("peer_id" => peer_id.to_string(), "connection_id" => format!("{connection_id}")));
```

Comment:
I think an unadorned variable will use the `Display` impl, won't it?

```suggestion
            .new(slog::o!("peer_id" => peer_id.to_string(), "connection_id" => connection_id));
```

Reply:
Thanks Michael! I tried that but I got interior mutability errors, do you know if the [`o!`](https://docs.rs/slog/latest/slog/macro.o.html) macro actually supports `%` and `#`? I see it in [`log!`](https://docs.rs/slog/latest/slog/) but [`o!`](https://docs.rs/slog/latest/slog/macro.o.html) is different  from [`log!`](https://docs.rs/slog/latest/slog/) descendants